sport/app/football/feed/Competitions.scala

```diff
@@ -26,7 +26,7 @@ trait Competitions extends implicits.Football {
       competitions.filter(_.url == path),
     )
 
-  def competitionsWithTag(tag: String): Option[Competition] = competitions.find(_.url.endsWith(tag))
+  def competitionsWithTag(tag: String): Option[Competition] = competitions.find(_.url.endsWith(s"/$tag"))
```

Comment:
Maybe we'd prefer to hardcode the football prefix into this check. Happy to discuss.

```suggestion
  def competitionsWithTag(tag: String): Option[Competition] = competitions.find(_.url == s"football/$tag")
```